Gemfile

```diff
@@ -7,10 +7,6 @@ if !Gem.win_platform? && RUBY_ENGINE == "ruby"
   gem 'byebug', group: [:development, :test]
 end
 
-if (Gem::Version.new(RUBY_VERSION) <=> Gem::Version.new("2.2.6")) < 0
-  gem 'rbnacl', '< 4.0'
-end
```

Comment:
Wasn't sure what this was about specifically. Let me know if it needs to be preserved.

Reply:
I think rbnacl 4 required ruby 2.2.6 so on earlier ruby-s we've installed rbnacl 3